migrations/app/20170912102542_app_users.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('users', function (table) {
+    table.increments('id')
+    table.string('username').unique().notNullable()
+    table.string('forename')
```

Comment:
Will we definitely be able to get `forename` and `surname`? And if so, what will they be used for?

Reply:
When I asked Tom this, his reply was that their existing system does contain forename an username and we should probably include it too. In our prototype there isn't a way to specify first and last names. One possible use would be for showing who you logged in as or in the case of workload point calculation the user who last updated the workloads points.